src/components/Calculator.js

```diff
@@ -1,33 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 
-const Calculator = () => (
-  <section className="calculator-section">
-    <div className="answer-section">
-      <span>0</span>
-    </div>
+const Calculator = () => {
+  const [details, setDetails] = useState({ total: '0', next: null, operation: null });
 
-    <div className="buttons-section">
-      <span className="button">Ac</span>
-      <span className="button">+/=</span>
-      <span className="button">%</span>
-      <span className="button operator">/</span>
-      <span className="button">7</span>
-      <span className="button">8</span>
-      <span className="button">9</span>
-      <span className="button operator">x</span>
-      <span className="button">4</span>
-      <span className="button">5</span>
-      <span className="button">6</span>
-      <span className="button operator">-</span>
-      <span className="button">1</span>
-      <span className="button">2</span>
-      <span className="button">3</span>
-      <span className="button operator">+</span>
-      <span className="button item17">0</span>
-      <span className="button">.</span>
-      <span className="button operator">=</span>
-    </div>
-  </section>
-);
+  const { total, next, operation } = details;
+
+  const makeCalculation = (btn) => {
+    setDetails((details) => calculate(details, btn.target.textContent));
+  };
+
+  return (
+    <section className="calculator-section">
+      <div className="answer-section">
+        <span>
+          {total}
+          {next}
+          {operation}
+        </span>
+      </div>
+
+      <div className="buttons-section">
+        <span className="button" onClick={makeCalculation}>Ac</span>
+        <span className="button" onClick={makeCalculation}>+/=</span>
+        <span className="button" onClick={makeCalculation}>%</span>
```

Comment:
- [ ] The button for plus or minus is expected to have `+/-` and not `+/=`. Kindly make the switch. 👍🏽